llvmlite/ir/values.py

```diff
@@ -52,6 +52,24 @@ def bitcast(self, typ):
                                                typ)
         return FormattedConstant(typ, op)
 
+    def ptrtoint(self, typ):
+        """
+        Cast this pointer constant to the given integer type.
+        """
+        if not isinstance(typ, types.IntType):
+            raise TypeError(
+                "can only ptrtoint() to integer type, not '%s'"
+                % (typ,))
+        if not isinstance(self.type, types.PointerType):
+            raise TypeError(
+                "can only call inttoptr() on pointer type, not '%s'"
```

Comment:
I know this is merged now, but in resolving conflicts for #735, I saw this and found it confusing. Maybe I'm not understanding something, but should this have been `ptrtoint` rather than `inttoptr`?

Reply:
yes!

Reply:
Fixed in https://github.com/numba/llvmlite/pull/794